satpy/readers/camel_l3_nc.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Reader for CAMEL Level 3 emissivity files in netCDF4 format.
+
+For more information about the data, see: <https://lpdaac.usgs.gov/products/cam5k30emv002/>.
+
+NOTE: This reader only supports the global 0.05 degree grid data.
+"""
+
+
+import datetime as dt
+import logging
+
+import xarray as xr
+from pyresample import geometry
+
+from satpy.readers.file_handlers import BaseFileHandler
+
+logger = logging.getLogger(__name__)
+
+# Area extent for the CAMEL product (global)
+GLOB_AREA_EXT = [-180, -90, 180, 90]
+
+
+class CAMELL3NCFileHandler(BaseFileHandler):
+    """File handler for Himawari L2 NOAA enterprise data in netCDF format."""
+
+    def __init__(self, filename, filename_info, filetype_info):
+        """Initialize the reader."""
+        super().__init__(filename, filename_info, filetype_info)
+        self.nc = xr.open_dataset(self.filename,
+                                  decode_cf=True,
+                                  mask_and_scale=True,
+                                  chunks={"xc": "auto", "yc": "auto"})
+
+        if self.nc.attrs["geospatial_lon_resolution"] != "0.05 degree grid ":
+            raise ValueError("Only 0.05 degree grid data is supported.")
+        if self.nc.attrs["geospatial_lat_resolution"] != "0.05 degree grid ":
+            raise ValueError("Only 0.05 degree grid data is supported.")
```

Comment:
It might be safer not to be so precise with comparison. Either do `"0.05" not in` or at least `.strip()` the attr string to remove extra whitespace.